(DOCS-2753) Remove instances of 'we' from docs (#12609)
DataDog/documentation: content/en/synthetics/guide/reusing-browser-test-journeys.md

  {{< img src="synthetics/guide/reusing-browser-test-journeys/dashboard_test_recording.mp4" alt="Recording parent test" video="true"  width="100%">}}

>  In the above example, it is checking to ensure that after logging into a test Datadog account, users are able to create a timeboard and that the timeboard is associated with the user who created it.

Reviewer: ```suggestion
 In the example above, the login subtest ensures that after logging into a Datadog test account, users can create a timeboard. That timeboard is then associated with the user.
```
